app/views/declaration/additional_fiscal_references.scala.html

```diff
@@ -60,13 +60,10 @@
                     <tr id="removable_elements__row@index">
                         <td id="removable_elements-row@index-label">@{elem}</td>
                         <td id="removable_elements-row@index-remove_button">
-                            <input
-                                type="submit"
-                                class="button--secondary"
-                                formmethod="post"
-                                formaction="@routes.AdditionalFiscalReferencesController.removeReference(mode, itemId, elem)"
-                                value="@messages("site.remove")"
-                            >
+                            <button class="remove button--secondary" type="submit" formmethod="post" formaction="@routes.AdditionalFiscalReferencesController.removeReference(mode, itemId, elem)">
+                                <span aria-hidden="true">@messages("site.remove")</span>
+                                <span class="visuallyhidden">@{messages("declaration.additionalFiscalReferences.remove.hint", elem)}</span>
+                            </button>
```

Comment:
Nit: It might be worth considering to create a dedicated template for secondary buttons?

Reply:
We already have the remove_button template that I tried to use here but that got rejected.  This 'button' is unique in that it defines a formaction.